pkg/release/release.go

```diff
@@ -33,3 +42,30 @@ var Image = map[string]string{
 	"pod":                       "quay.io/microshift/pause:" + Base,
 	"service_ca_operator":       "quay.io/microshift/service-ca-operator:" + Base,
 }
+
+func init() {
+	arch_replacer := strings.NewReplacer("amd64", "x86_64", "arm64", "aarch64")
```

Comment:
I wonder why arch names have been replaced: `amd64` to `x86_64` and `arm64` to `aarch64`

Reply:
My motivation for this was user experience. The x86_64/aarch64 architecture names are more idiomatic on Linux systems in general and RPM-based distros in particular.

For example, this way users can use `$(uname -i)` in scripts and avoid conditional logic and substitutions as with the Golang names. It’s also consistent with the RPM package naming scheme.